geoips_plugin_example/__init__.py

```diff
@@ -9,3 +9,4 @@
 # # # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the included license
 # # # for more details. If you did not receive the license, for more information see:
 # # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
+"""Init file for geoips_plugin_example."""
```

Comment:
@lance-wilson This needs to have the "version" information like is contained in the main geoips repo __init__.py file.  Could you also please update the Issue to note that the version imports should be added to the top level __init__.py file?

Additionally, every directory that contains python files in any sub-directory must have an __init__.py file.  Maybe make a note of that in the main issue as well?

Reply:
Done.

Reply:
In [1]: import geoips_plugin_example

In [2]: geoips_plugin_example.__version__
Out[2]: '1.6.0.post24'

Looks good